apps/playground/src/app/modules/components/checkbox-module/checkbox-module.component.ts

```diff
@@ -2,54 +2,22 @@
  * Angular imports
  */
 import { Component, OnInit } from '@angular/core';
-import { FormBuilder, FormGroup } from '@angular/forms';
+import { FormControl, FormGroup, NonNullableFormBuilder, Validators } from '@angular/forms';
+
+/**
+ * 3rd-party imports
+ */
 import { ElementSize } from '@betagouv/ngx-dsfr';
 
```

Comment:
I have mistakes in the Template because Typed Forms is not correctly implemented.

You should add this here : 
```typescript
/**
 * TypeScript entities and constants
 */
interface SettingsForm {
  legend: FormControl<string>;
  hint: FormControl<string>;
  inline: FormControl<boolean>;
  disabled: FormControl<boolean>;
  size: FormControl<ElementSize>;
  hasFailed: FormControl<boolean>;
  failureMessage: FormControl<string>;
  hasSucceeded: FormControl<boolean>;
  successMessage: FormControl<string>;
}

interface SimulationForm {
  checkbox: FormControl<string[]>;
}
```